src/accelerate/hooks.py

```diff
@@ -30,6 +30,9 @@
 from .utils.other import recursive_getattr
 
 
+_accelerate_added_attributes = ["to", "cuda", "npu", "xpu", "mlu"]
```

Comment:
I would have hoped we could reuse this constant when the hooks are added (I guess [here](https://github.com/huggingface/accelerate/blob/f62854a28167a8775e8808669e0ec5f4a1f28537/src/accelerate/big_modeling.py#L460-L468)) but that won't quite work because of the additional checks needed there. But we could still add a comment there that if a new warning hooks is added, `_accelerate_added_attributes` should be updated.

Reply:
Good idea ! Thanks !